metadata.json

```diff
@@ -9,12 +9,12 @@
   "issues_url": "https://github.com/crayfishx/puppet-rsyslog/issues",
   "dependencies": [
     {"name":"puppetlabs-stdlib","version_requirement":">= 1.0.0 < 5.0.0"},
-    {"name":"puppetlabs-concat","version_requirement":">= 2.0.0 < 3.0.0"}
+    {"name":"puppetlabs-concat","version_requirement":">= 2.0.0 < 5.0.0"}
   ],
   "requirements": [
     { 
       "name": "puppet",
-      "version_requirement": ">= 4.0.0 < 5.0.0"
+      "version_requirement": ">= 4.4.0 < 6.0.0"
```

Comment:
I believe the only reason I ran into this during testing. If I remember correctly, is that both your existing tests and my new tests were failing in Puppet < 4.4. I'll play with that today and figure out why.

I know that "official" Puppet 4 support starts with Puppet >= 4.7.0, so maybe there is something ruby or API related that isn't technically "breaking", but I'm not sure.